netbeans/CheckerFrameworkPlugin/release/checkerstrings.properties

```diff
@@ -0,0 +1,18 @@
+org.checkerframework.checker.nullness.NullnessChecker=Nullness Checker
+org.checkerframework.checker.interning.InterningChecker=Interning Checker
+org.checkerframework.checker.lock.LockChecker=Lock Checker
+org.checkerframework.checker.fenum.FenumChecker=Fenum Checker
+TaintingChecker=Tainting Checker
```

Comment:
In the checker framework documentation, it says "to run the Tainting Checker, supply the -processor TaintingChecker command-line option to javac" so I didn't add a package.